src/mscorlib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -777,6 +775,66 @@ public int EnsureCapacity(int capacity)
             return newSize;
         }
 
+        /// <summary>
+        /// Sets the capacity of this dictionary to hold up at least all existing entries
```

Comment:
This is not strictly true is it? It picks a larger prime.
How about "Sets the capacity of this dictionary to what it would be if it had been originally initialized with all its entries"